src/app/_components/PageHeader.tsx

```diff
@@ -40,7 +40,7 @@ export function PageHeader({
       {isFeatured && <SectionDivider title="Featured" />}
       <div className="flex flex-col gap-6 md:flex-row">
         <div className="flex flex-col gap-4 md:w-1/2">
-          <Heading tag="h1" variant="4xl" className="text-balance">
+          <Heading tag="h1" variant="4xl">
```

Comment:
Have you checked the difference between `text-balance` and `text-pretty`?

Reply:
We should use `text-balance` on headings and `text-pretty` on paragraphs 👍. [Source](https://blog.stephaniestimac.com/posts/2023/10/css-text-wrap/#:~:text=Use%20text%2Dwrap%3A%20balance%3B,good%20candidate%20for%20progressive%20enhancement.).

Reply:
Interesting! Thank you for sharing.